week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,17 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+function isStudent(student){
```

Comment:
Actually the parameter here is coming from the group array, so you don't know if it's a student yet. A more appropriate parameter name might just be `member`